pkg/kbld/cmd/image.go

```diff
@@ -24,6 +24,30 @@ func (imgs Images) ForImage(url string) (Image, bool) {
 	return Image{}, false
 }
 
+func (i Image) Equal(other Image) bool {
+	if i.URL != other.URL {
```

Comment:
i think we should move this into imageStruct object (https://github.com/k14s/kbld/blob/2c00de6b7df30b5214c797c1e91577d81b86031d/pkg/kbld/cmd/image.go#L37). comparison could be simplfied as well to something like this: `i.URL == other.URL && reflect.DeepEqual(i.Metas, other.Metas)`

Reply:
Sounds good. Done.